src/test/java/com/fundation/webservice/model/CriteriaAudioTest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.webservice.model;
+
+/**
```

Comment:
Relocate this comment block just before class definition.

Reply:
fixed